basic/src/test/java/org/jboss/hal/testsuite/test/configuration/remoting/endpointconfiguration/EndpointConfigurationAttributesTestCase.java

```diff
@@ -37,17 +40,35 @@ public void editBufferRegionSize() throws Exception {
     }
 
     @Test
+    public void editBufferRegionSizeInvalidValue() throws Exception {
+        final String invalidBufferRegionSize = "invalid_buffer_region_size";
+        ConfigFragment config = page.getConfigFragment();
+        config.editTextAndSave(BUFFER_REGION_SIZE, invalidBufferRegionSize);
+        Assert.assertTrue("There should be validation error", config.isErrorShownInForm());
+        config.cancel();
```

Comment:
For this kind of scenarios we usually use `ConfigChecker.Builder#verifyFormNotSaved()`. But this is OK.